packages/babel-plugin/src/keyframes/__tests__/tagged-template.test.tsx

```diff
@@ -0,0 +1,840 @@
+import { transform } from '../../__tests__/test-utils';
```

Comment:
I haven't reviewed all the test cases yet, so forgive me if you already addressed these cases...just wondering what happens if
- we have a keyframe defined multiple times ( eg https://developer.mozilla.org/en-US/docs/Web/CSS/@keyframes#when_a_keyframe_is_defined_multiple_times)
- we have multiple animations ( eg https://css-tricks.com/almanac/properties/a/animation/#multiple-animations)

Reply:
Added tests for these cases